xchange-tradeogre/src/main/java/org/knowm/xchange/tradeogre/TradeOgreAdapters.java

```diff
@@ -23,7 +23,7 @@
 public class TradeOgreAdapters {
 
   public static String adaptCurrencyPair(CurrencyPair currencyPair) {
-    return currencyPair.counter.toString() + "-" + currencyPair.base.toString();
```

Comment:
please use getters instead of direct field access

Reply:
I'm using it on a project, and this just the change I need, I'll try to fix tests and use getters.